common/test/assets/AssetsTest.scala

```diff
@@ -13,4 +13,11 @@ class AssetsTest extends FlatSpec with Matchers with OneAppPerSuite {
     static("zen3").toString should be("simon says" + "the quieter you become the more you are able to hear.")
   }
 
+  "CssMap" should "collect css maps" in {
+    val static = new CssMap("assets/testclass-map.json", true)
```

Comment:
... using `true` here forces people to build their assets a certain way just to pass tests. You should be able to test the `CssMap` in this situation by extending the class, and overriding the bits you need to stub.